src/open_inwoner/configurations/tests/test_analytics.py

```diff
@@ -111,3 +115,57 @@ def test_siteimprove_analytics_not_rendered(self):
 
         self.assertTemplateNotUsed(response, "analytics/siteimprove.html")
         self.assertNotContains(response, "<!-- SiteImprove -->")
+
+
+@tag("e2e")
+@override_settings(ROOT_URLCONF="open_inwoner.cms.tests.urls")
+class CookieBannerPlaywrightTests(PlaywrightSyncLiveServerTestCase):
+    def setUp(self):
+        from open_inwoner.accounts.tests.factories import DigidUserFactory
+
+        super().setUp()
+
+        self.user = DigidUserFactory.create()
+        self.user_login_state = self.get_user_bsn_login_state(self.user)
+
+    def test_cookie_accept(self):
+        config = SiteConfiguration.get_solo()
+
+        context = self.browser.new_context(storage_state=self.user_login_state)
+
+        page = context.new_page()
+        page.goto(self.live_reverse("profile:action_list"))
+
+        expect(page.get_by_text(config.cookie_info_text)).to_be_visible()
+
+        accept_button = page.get_by_text(_("Alles toestaan"))
+        accept_button.click()
```

Comment:
We have an ongoing discussion if we should do a more specific check to see if a page contains a certain string instead of just checking the global page. 

So maybe we can locate the actual popup element, check if it visible and then check the text and buttons chained from that (you can call `.locator()`/get_by_xyz etc on a Locator to make more specific locations).